src/sagemaker/workflow/steps.py

```diff
@@ -111,10 +112,14 @@ def __init__(
             name (str): The name of the training step.
             estimator (EstimatorBase): A `sagemaker.estimator.EstimatorBase` instance.
             inputs (TrainingInput): A `sagemaker.inputs.TrainingInput` instance. Defaults to `None`.
+            experiment_config (Dict[str, str]): A dictionary describing the experiment configuration
+                for the training job. Dictionary contains three optional keys,
+                'ExperimentName', 'TrialName', and 'TrialComponentDisplayName'. Default: to `None`.
```

Comment:
```suggestion
                'ExperimentName', 'TrialName', and 'TrialComponentDisplayName'. Defaults: to `None`.
```

Reply:
Changed to "Defaults to" as per other comments.